packages/firestore/src/protos/firestore_bundle_proto.ts

```diff
@@ -54,6 +54,9 @@ export interface BundledDocumentMetadata {
 
   /** BundledDocumentMetadata exists */
   exists?: boolean | null;
+
+  /** The names of the queries in this bundle that this document matches to. */
+  queries?: string[] | null;
```

Comment:
In Proto3, is not possible to have a missing array. Missing arrays are represented as empty lists, so this should be `queries: string[]`.

Reply:
Done.